daemon/nextbox_daemon/api/generic.py

```diff
@@ -165,6 +165,23 @@ def poweroff():
     return success(data={})
 
 
+@generic_api.route("/debianVersion", methods=["GET"])
+@requires_auth
+def debianVersion():
+    log.info("debianVersion - by /debianVersion request")
+    cr = CommandRunner("cat /etc/debian_version")
```

Comment:
please don't overuse shell and the `CommandRunner` if it's not needed - that's a plain text file, right? So just open/read the file using python, which is less error prone and better controllable.

Reply:
done